conf.py

```diff
@@ -4,8 +4,8 @@
     'LOG_DIR': 'logs',
     'GAMES_DIR': 'games',
     'BEST_MODEL': 'best_model.h5',
-    'SHOW_EACH_MOVE': False,
-    'SHOW_END_GAME': False,
+    'SHOW_EACH_MOVE': True,
```

Comment:
You probably should not change those. I feel someone pulling the code and running does not want to see all these logs.